projects/knora-ui/src/lib/viewer/values/color-value/color-value.component.html

```diff
@@ -1,27 +1,28 @@
-<span [formGroup]="form">
-    <mat-form-field class="large-field color-field child-value-component" floatLabel="never">
-        <kui-color-picker #colorInput [formControlName]="'colorValue'" class="value" placeholder="Color value"
-            [readonly]="mode === 'read'" [errorStateMatcher]="matcher">
-        </kui-color-picker>
-        <mat-error *ngIf="valueFormControl.hasError('valueNotChanged')">
-            Please change the value.
-        </mat-error>
-        <mat-error *ngIf="valueFormControl.hasError('pattern')">
-            Please enter a hex color value.
-        </mat-error>
-    </mat-form-field>
-    <mat-form-field class="large-field value-component-comment" *ngIf="shouldShowComment">
-        <textarea matInput
-                  cdkTextareaAutosize
-                  #autosize="cdkTextareaAutosize"
-                  cdkAutosizeMinRows="1"
-                  cdkAutosizeMaxRows="6"
-                  [formControlName]="'comment'"
-                  class="comment"
-                  placeholder="Comment"
-                  type="text"
-                  [readonly]="mode === 'read'"
-                  spellcheck="false">
-          </textarea>
-      </mat-form-field>
-</span>
\ No newline at end of file
+<div *ngIf="mode === 'read'; else showForm" class="read-mode-view">
+    <p>Hex value:</p>
+    <p class="rm-value">{{ valueFormControl.value }}</p>
+    <p>Preview:</p>
+    <div class="color-preview" [style.background-color]="valueFormControl.value"></div>
+    <p class="rm-comment" *ngIf="shouldShowComment">{{ commentFormControl.value }}</p>
+</div>
+<ng-template #showForm>
+    <span [formGroup]="form">
+        <mat-form-field class="large-field color-field child-value-component" floatLabel="never">
+            <kui-color-picker #colorInput [formControlName]="'colorValue'" class="value" placeholder="Color value"
+                [readonly]="mode === 'read'" [errorStateMatcher]="matcher">
+            </kui-color-picker>
+            <mat-error *ngIf="valueFormControl.hasError('valueNotChanged')">
+                Please change the value.
+            </mat-error>
+            <mat-error *ngIf="valueFormControl.hasError('pattern')">
+                Please enter a hex color value.
+            </mat-error>
+        </mat-form-field>
+        <mat-form-field class="large-field value-component-comment">
+            <textarea matInput cdkTextareaAutosize #autosize="cdkTextareaAutosize" cdkAutosizeMinRows="1"
```

Comment:
I think it would be helpful if the code formatting is the same for all templates (see boolean value template).

Reply:
done in 37865b7